.storybook/preview.js

```diff
@@ -1,9 +1,22 @@
+import colors from '../src/styles/colors';
+
 export const parameters = {
-  actions: { argTypesRegex: "^on[A-Z].*" },
-  controls: {
-    matchers: {
-      color: /(background|color)$/i,
-      date: /Date$/,
-    },
-  },
-}
\ No newline at end of file
+	actions: { argTypesRegex: '^on[A-Z].*' },
+	controls: {
+		matchers: {
+			color: /(background|color)$/i,
+			date: /Date$/,
+		},
+	},
+	backgrounds: {
+		default: 'dark',
+		values: [
+			{ name: 'dark', value: '#111' },
+			{ name: 'light', value: '#F8F8F8' },
+			{ name: 'black', value: '#000000' },
+			{ name: 'white', value: '#FFFFFF' },
+			{ name: 'purple', value: colors.gradients.purple },
+			{ name: 'darkBlue', value: colors.gradients.darkBlue },
+		],
```

Comment:
Added some common backgrounds on the toolbar to test our components. It includes common colors from our designs.